tsconfig.base.json

```diff
@@ -16,6 +16,9 @@
     "baseUrl": ".",
     "paths": {
       "@api/*": ["apps/cugetreg-api/src/*"],
+      "@cugetreg/codegen": ["libs/codegen/src"],
+      "@cugetreg/codegen/future": ["libs/codegen/src/future"],
```

Comment:
It would be great if we named the alias path `@lib/something`

Reply:
I agree. Something like `@libs/codegen` would be much easier to use.